client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,24 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
+  return (
```

Comment:
what you could also have considered is moving the object destructuring to here so it's less code